docs/content/faq/execution.md

```diff
@@ -24,3 +24,24 @@ DLT-META translates input metadata into Delta table as DataflowSpecs
 **Q. How many DLT pipelines will be launched using DLT-META?**
 
 DLT-META uses data_flow_group to launch DLT pipelines, so all the tables belongs to same group will be executed under single DLT pipeline. 
+
+**Q. Can we run onboarding for bronze layer only?**
+
+Yes! Remove silver related attributes from onboarding file and call  ```onboard_bronze_dataflow_spec()``` API from ```OnboardDataflowspec```. Similarly you can run silver layer onboarding separately using ```onboard_silver_dataflow_spec()``` API from ```OnboardDataflowspec``` with silver parameters included in ```onboarding_params_map```
```

Comment:
triple back quotes are for code on a separate lines, just use single back quotes